.circleci/config.yml

```diff
@@ -213,10 +211,17 @@ workflows:
 orbs:
   slack: circleci/slack@3.3.0
   build_containers_and_push_to_ecr:
+    orbs:
+      docker: circleci/docker@1.4.0
     commands:
+      dockerhub_login:
```

Comment:
Do the DOCKER_USER etc in this step need to be vars (so $)?

Reply:
The docker circle CI orb recognises the values as environment var names so we just have to pass that and it works it out. see: https://circleci.com/developer/orbs/orb/circleci/docker#commands-check  (expand the panel to see the code) and https://circleci.com/docs/2.0/reusing-config/#environment-variable-name

Reply:
the others do need the login details with a $ as per the docker auth pull docs in circle ci: https://circleci.com/docs/2.0/private-images/